docs/manual/build_system/prerequisites.qbk

```diff
@@ -69,6 +69,9 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
         [Used for OS-thread pinning and NUMA awareness. This library is optional on Mac OSX.]]
 ]
 
+[important Due to lack of Thread Local Storage support it is advised to 
+        use Boost V1.59.0 when compiling __hpx__ over OSX platform.]
+      
```

Comment:
May I suggest to write which versions do not work instead? @zao wrote that for him Boost 1.58 and 1.60 failed to work, all others were fine. I also think that this limitation is true only if using clang/libc++ on OSX. I think this needs to be added as well.